### PR TITLE
Add new commands Makefile

### LINKED ISSUE #16: Add commands on Makefile
We want a commands as following
```
## -- Tools 🔧 -----------------------------------------------------------------
bash: ## Bash into php container
	$(EXEC_PHP) bash
```

### PR DESCRIPTION
**Pull request by yalefresne**
Close #16

### CODE REVIEW

**Comment by senzowayne** [comment]
Oh que c'est beau c'est là que je dois m'inspirer 😜

File: Makefile [hunk 1]
```diff
@@ -54,10 +56,23 @@ warmup: ## Warmup the cache
 purge: ## Purge cache and logs
 	rm -rf var/cache/* var/logs/*
 
+## —— Doctrine ▶️ ———————————————————————————————————————————————————————————————
+dgm: ## Generate a blank migration class
+	$(SYMFONY) doctrine:migrations:generate
```
Comment (Rantignac) [inline Makefile:61]:
```suggestion
	$(SYMFONY) d:m:g
```

File: Makefile [hunk 2]
```diff
@@ -54,10 +56,23 @@ warmup: ## Warmup the cache
...
+dmm: ## Execute migrations
+	$(SYMFONY) doctrine:migrations:migrate
```
Comment (Rantignac) [inline Makefile:63]:
```suggestion
	$(SYMFONY) d:m:m
```

File: Makefile (hunk 1 above)
Comment (yalefresne) [reply Makefile:61]: I wanted to be explicit on Makefile. Shorter command is nice when you must to type it.

File: Makefile (hunk 2 above)
Comment (yalefresne) [reply Makefile:63]: I wanted to be explicit on Makefile. Shorter command is nice when you must to type it.

**Comment by yalefresne** [comment]
@senzowayne I myself was inspired by this [https://www.strangebuzz.com/en/blog/setting-a-ci-cd-workflow-for-a-symfony-project-thanks-to-the-github-actions](https://www.strangebuzz.com/en/blog/setting-a-ci-cd-workflow-for-a-symfony-project-thanks-to-the-github-actions)

File: Makefile (hunk 1 above)
Comment (Rantignac) [reply Makefile:61]: In this case don't use a shorter command for the clear cache

Comment (yalefresne) [reply Makefile:61]: You're right, i have changed clear cache command to be explicit too.

**Review by Rantignac — APPROVED** [review]

**Pull request merged**